layers/blog/components/global/node-blog-full.vue

```diff
@@ -1,25 +1,16 @@
 <template>
-    <div class="node">
-      <h2 v-if="title">Blog: {{ title }}</h2>
-      <slot name="image">
-        <component :is="useDrupalCe().renderCustomElements($attrs.image)" />
-      </slot>
-      <slot name="body">
-        <component :is="useDrupalCe().renderCustomElements($attrs.body)" />
-      </slot>
-    </div>
-  </template>
-  
-  <script setup lang="ts">
-  defineSlots<{
-    body();
-    image();
-  }>()
-  defineProps<{
-    title?: string;
-    type?: string;
-    created?: number | string;
-    // Layout-builder support.
-    sections?: object;
-  }>()
-  </script>
\ No newline at end of file
+  <div class="node">
+    <h2 v-if="title">Blog: {{ title }}</h2>
+    <component :is="useDrupalCe().renderCustomElements(sections)" />
+  </div>
+</template>
+
+<script setup lang="ts">
+const { getPage } = useDrupalCe();
+defineProps<{
+  // Layout-builder support.
+  sections?: object;
+}>()
+
+const title = getPage().value.title;
```

Comment:
this is weird, why not stay with the title prop as is?
imho the previous props/slots could be kept, but nevermind. it's just an example atm.

But for making it proper we should configure props for all the attributes we want to pass through, and having at least the title in there makes sense.

Reply:
The title prop is not output in the node layout.